src/components/FarmCard/FarmCard.tsx

```diff
@@ -1,6 +1,7 @@
 import React, { useState } from 'react';
 import { Box, Typography, useMediaQuery } from '@material-ui/core';
 import { makeStyles, useTheme } from '@material-ui/core/styles';
+import cx from 'classnames';
```

Comment:
What is this classnames?

Reply:
I use `classnames` in order to merge two classes